ddtrace/tracer.py

```diff
@@ -609,14 +610,17 @@ def _on_span_finish(self, span):
 
     def _initialize_span_processors(self, appsec_enabled=asbool(os.getenv("DD_APPSEC_ENABLED", default=False))):
         # type: (Optional[bool]) -> None
+        for sp in self._span_processors:
```

Comment:
should we add a test case that existing span processors get shutdown and removed?

Reply:
True, good idea!